Source/WebInspectorUI/UserInterface/Controllers/WebInspectorExtensionController.js

```diff
@@ -197,6 +197,17 @@ WI.WebInspectorExtensionController = class WebInspectorExtensionController exten
         return target.PageAgent.reload.invoke({ignoreCache});
     }
 
+    loadURLForExtension(extensionTabID, sourceURL)
```

Comment:
NIT: The naming of this makes me think it's gonna do a `fetch` or something.  How about `navigateTabForExtension` (to match `createTabForExtension` naming)?

NIT: `sourceURL` has a kinda specific meaning in Web Inspector (specifically `//# sourceURL=...`).  Can we just use `url`?

Reply:
Using `navigate` makes sense to me.

The ship has sailed for `sourceURL`, it is used in a couple other places in this Inspector Extensions code already.